src/ui.js

```diff
@@ -25,7 +25,7 @@ export const makeTable = async () => {
     row.push(
       company[COMPANY_NAME_FIELD_NAME],
       company[STATUS_FIELD_NAME],
-      company[CREATED_AT_FIELD_NAME],
+      `${String(new Date(company[CREATED_AT_FIELD_NAME]).getUTCHours()).padStart(2, 0)}:${String(new Date(company[CREATED_AT_FIELD_NAME]).getUTCMinutes()).padStart(2, 0)}`,
```

Comment:
I think it works.

One thing - this line of code is extremely hard to read. A very good improvement would be to separate it into multiple lines of code. Ideally, write a new function that takes unformatted date and returns the formatted one.

Really loved it that you preserved UTC timezone. Good job!